ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.html

```diff
@@ -1,33 +1,36 @@
-<span class="activiti-label mdl-badge"
+<span class="activiti-label mdl-badge" id="people-title"
       [attr.data-badge]="people?.length">{{ 'TASK_DETAILS.LABELS.PEOPLE' | translate }}</span>
-<div id="addPeople" (click)="showDialog()" class="icon material-icons">add</div>
-<div class="mdl-tooltip" for="addPeople">
+<div *ngIf="!readOnly" id="addPeople" (click)="showDialog()" class="icon material-icons">add</div>
+<div *ngIf="!readOnly" class="mdl-tooltip" data-mdl-for="addPeople">
     Add a person
 </div>
 <div class="menu-container" *ngIf="people?.length > 0">
     <ul class='mdl-list'>
         <li class="mdl-list__item" *ngFor="let user of people">
             <span class="mdl-list__item-primary-content">
-            <i class="material-icons mdl-list__item-icon">face</i>
-            {{user.firstName}}
+                <i class="material-icons mdl-list__item-icon">face</i>
+                <span id="user-{{user.id}}">{{user.firstName}} {{user.lastName}}</span>
             </span>
+            <a *ngIf="!readOnly" class="mdl-list__item-secondary-action">
+                <i id="remove" class="material-icons"
+                   (click)="removeInvolvedUser(user)">delete</i>
+            </a>
         </li>
     </ul>
 </div>
-<div *ngIf="people?.length === 0">
+<div *ngIf="people?.length === 0" id="no-people-label">
     {{ 'TASK_DETAILS.PEOPLE.NONE' | translate }}
 </div>
 
-<dialog class="mdl-dialog" #dialog>
-    <h4 class="mdl-dialog__title">New User</h4>
-    <div class="mdl-dialog__content">
-        <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-            <input class="mdl-textfield__input" type="text" id="people" />
-            <label class="mdl-textfield__label" for="people">Name</label>
-        </div>
+<dialog class="mdl-dialog" id="add-people-dialog" #dialog>
+    <h4 class="mdl-dialog__title" id="add-people-dialog-title">Involve User</h4>
+    <div class="mdl-dialog__content involve-user-padding">
```

Comment:
Not very informative class name, I recommend changing `involve-user-padding` to something like `add-people-dialog__content`